lib/portlayer/exec/exec.go

```diff
@@ -169,6 +162,15 @@ func eventCallback(ie events.Event) {
 				}()
 			case StateRemoved:
 				log.Debugf("Container(%s) %s via event activity", container, newState)
+				if container.vm != nil {
+					isFixing := container.vm.Fixing.Load()
+					if isFixing != nil && isFixing.(bool) {
+						// is fixing vm, which will be registered back soon, so do not remove from containers cache
+						log.Debugf("Container(%s) %s is being fixed", container.ExecConfig.ID)
+						container.vm.Fixing.Store(false)
```

Comment:
Why are we setting it to false here? Shouldn't it be set to false in `fixVM` after the fixing is done?